api/v1alpha6/openstackclustertemplate_types.go

```diff
@@ -44,6 +44,8 @@ type OpenStackClusterTemplate struct {
 	Spec OpenStackClusterTemplateSpec `json:"spec,omitempty"`
 }
 
+// +genclient
+// +genclient:Namespaced
```

Comment:
This was not rendered: https://deploy-preview-1702--kubernetes-sigs-cluster-api-openstack.netlify.app/api/v1alpha6/api